Tests/XCUITests/LoginsTests.swift

```diff
@@ -72,22 +86,35 @@ class SaveLoginTest: BaseTestCase {
         XCTAssertEqual(app.tables["Login List"].cells.count, defaultNumRowsLoginsList + 1)
     }
 
+    // https://testrail.stage.mozaws.net/index.php?/cases/view/52768
     // Smoketest
     func testSaveLogin() {
         closeURLBar()
         // Initially the login list should be empty
-        openLoginsSettings()
+        openLoginsSettingsFromBrowserTab()
```

Comment:
With this I'm trying to go to Logins via BrowserTabMenu and not from Settings. The test is flaky because we need to swipe up the settings view to get to logins... hopefully entering the menu this way the test will pass more consistently.